classes/Validate.php

```diff
@@ -661,7 +661,11 @@ public static function isProductVisibility($s)
      */
     public static function isInt($value)
     {
-        return ((string)(int)$value === (string)$value || $value === false);
+        if (!is_object($value)) {
+            return ((string)(int)$value === (string)$value || $value === false);
+        }
+        
+        return false;
```

Comment:
@jocel1 could you please review my code, thanks in advance :)

Reply:
Why don't we use the native `is_int()` ?

Reply:
Because we do not want the same behaviour. While `Validate::isInt('2')` returns `true`, `is_int('2')` will say `false`